plugins/new/gatsby-node.js

```diff
@@ -0,0 +1,306 @@
+const path = require('path');
+const stream = require('stream');
+const { promisify } = require('util');
+const fs = require('fs').promises;
+const { transformBreadcrumbs } = require('../../src/utils/setup/transform-breadcrumbs.js');
+const { saveStaticFiles } = require('../../src/utils/setup/save-asset-files');
+const { siteMetadata } = require('../../src/utils/site-metadata');
+const { getNestedValue } = require('../../src/utils/get-nested-value');
+const pipeline = promisify(stream.pipeline);
+const got = require(`got`);
+const { parser } = require(`stream-json/jsonl/Parser`);
+const { sourceNodes } = require(`./other-things-to-source`);
+const _ = require(`lodash`);
+
+let isAssociatedProduct = false;
+let associatedReposInfo = {};
+let db;
+const isPreview = process.env.GATSBY_IS_PREVIEW === `true`;
+
+exports.createSchemaCustomization = async ({ actions }) => {
+  const { createTypes } = actions;
+  const typeDefs = `
+    type Page implements Node @dontInfer {
+      page_id: String
+      branch: String
+      pagePath: String
+      ast: JSON!
+    }
+
+    type SnootyMetadata implements Node @dontInfer {
+      metadata: JSON
+      branch: String
+    }
+
+    type RemoteMetadata implements Node @dontInfer {
+      remoteMetadata: JSON
+    }
+
+    type ChangelogData implements Node @dontInfer {
+      changelogData: JSON
+    }
+  `;
+  createTypes(typeDefs);
+};
+
+const saveFile = async (file, data) => {
+  await fs.mkdir(path.join('static', path.dirname(file)), {
+    recursive: true,
+  });
+  await fs.writeFile(path.join('static', file), data, 'binary');
+};
+
+let manifestMetadata;
+
+const APIBase = process.env.API_BASE || `https://snooty-data-api.mongodb.com`;
+
+exports.sourceNodes = async ({ actions, createNodeId, getNode, createContentDigest, cache }) => {
+  let hasOpenAPIChangelog = false;
+  const { createNode } = actions;
+
+  let pageCount = 0;
+  const fileWritePromises = [];
+  const lastFetched = (await cache.get(`lastFetched`)) || 0;
+  console.log({ lastFetched });
+  let url;
+  if (lastFetched) {
+    url = `${APIBase}/projects/${process.env.GATSBY_SITE}/documents`;
+  } else {
+    url = `${APIBase}/projects/${process.env.GATSBY_SITE}/documents?updated=${lastFetched}`;
+  }
```

Comment:
Forgot to switch the two around

```suggestion
  if (lastFetched) {
    url = `${APIBase}/projects/${process.env.GATSBY_SITE}/documents?updated=${lastFetched}`;
  } else {
    url = `${APIBase}/projects/${process.env.GATSBY_SITE}/documents`;
  }
```